backend/src/test/controller/PromotionController.test.ts

```diff
@@ -273,6 +273,72 @@ describe('Unit tests for PromotionController', function () {
     request(app).delete(`/promotions/${nonExistentUUID}`).expect(204, done);
   });
 
+  test('POST /promotions/:id/upVote', async (done) => {
+    const user: User = new UserFactory().generate();
+    const promotion = new PromotionFactory().generate(
+      user,
+      new DiscountFactory().generate(DiscountType.PERCENTAGE),
+      [new ScheduleFactory().generate()]
+    );
+
+    await userRepository.save(user);
+    await promotionRepository.save(promotion);
+
+    request(app)
+      .post(`/promotions/${promotion.id}/upVote`)
+      .expect(204)
+      .end((err) => {
+        if (err) return done(err);
+        return getManager().transaction(
+          'READ UNCOMMITTED',
+          async (transactionalEntityManager) => {
+            // check that promotion votes has incremented
+            const promotionRepository = transactionalEntityManager.getCustomRepository(
+              PromotionRepository
+            );
+            const newPromotion = await promotionRepository.findOneOrFail(
+              promotion.id
+            );
+            expect(newPromotion.votes).toEqual(1);
+            done();
+          }
+        );
+      });
+  });
+
+  test('POST /promotions/:id/downVote', async (done) => {
+    const user: User = new UserFactory().generate();
+    const promotion = new PromotionFactory().generate(
+      user,
+      new DiscountFactory().generate(DiscountType.PERCENTAGE),
+      [new ScheduleFactory().generate()]
+    );
+
+    await userRepository.save(user);
+    await promotionRepository.save(promotion);
+
+    request(app)
+      .post(`/promotions/${promotion.id}/downVote`)
+      .expect(204)
+      .end((err) => {
+        if (err) return done(err);
+        return getManager().transaction(
+          'READ UNCOMMITTED',
+          async (transactionalEntityManager) => {
+            // check that promotion votes has incremented
```

Comment:
```suggestion
            // check that promotion votes has decremented
```
Another minor change!

Reply:
Oopsies thanks for catching this!